pages/api/auth/[...nextauth].js

```diff
@@ -1,27 +1,30 @@
-import NextAuth from "next-auth"; // Next Authentication
-import Providers from "next-auth/providers"; // Next Authentication providers
+import NextAuth from 'next-auth'; // Next Authentication
+import Providers from 'next-auth/providers'; // Next Authentication providers
+// TODO retrieve custom ORM model using Prisma adapter
+// import Adapters from 'next-auth/adapters'; // Next Authentication adapters
+// import prisma from 'pages/index';
 
 // Database Configuration
 const databaseConfig = {
-  type: "postgres",
+  type: 'postgres',
   host: process.env.POSTGRES_HOST,
   port: 5432,
   username: process.env.POSTGRES_USER,
   password: process.env.POSTGRES_PASSWORD,
   database: process.env.POSTGRES_DBNAME,
   // Bypass SSL rejectUnauthorized (Heroku, etc)
   ssl: {
-    sslmode: "require",
+    sslmode: 'require',
     rejectUnauthorized: false,
   },
 };
 
 // Administrator emails
-const adminEmails = ["contact+admin@anishagnihotri.com"];
+const adminEmails = ['contact+admin@anishagnihotri.com', 'ericli@uwblueprint.org'];
 
 export default NextAuth({
   // Site URL
-  site: process.env.NEXTAUTH_URL || "http://localhost:3000",
+  site: process.env.SITE || 'http://localhost:3000',
```

Comment:
Will have to confirm, but I believe this should stay `process.env.NEXTAUTH_URL`.

Reply:
changing it back.